src/Catalyst.Core.Lib.Tests/UnitTests/Utils/JsonProtoObjectConverterTests.cs

```diff
@@ -0,0 +1,73 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using Catalyst.Core.Lib.Util;
+using Catalyst.Protocol.Wire;
+using Catalyst.TestUtils;
+using FluentAssertions;
+using Google.Protobuf;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Catalyst.Core.Lib.Tests.UnitTests.Utils
+{
+    public sealed class JsonProtoObjectConverterTests
+    {
+        private readonly string _transactionJson;
+        private readonly TransactionBroadcast _transaction;
+        private readonly JsonProtoObjectConverter<TransactionBroadcast> _jsonProtoObjectConverter;
+
+        public JsonProtoObjectConverterTests()
+        {
+            _jsonProtoObjectConverter = new JsonProtoObjectConverter<TransactionBroadcast>();
+            _transaction = TransactionHelper.GetContractTransaction(ByteString.CopyFromUtf8("test"), 10, 10, 10);
+            _transactionJson = "{ \"publicEntry\": { \"receiverAddress\": \"cmVjZWl2ZXI=\", \"senderAddress\": \"c2VuZGVy\", \"amount\": \"Cg==\", \"data\": \"dGVzdA==\", \"timestamp\": \"1970-01-01T03:25:45Z\", \"gasPrice\": \"Cg==\", \"gasLimit\": \"10\", \"transactionFees\": \"Ag==\", \"signature\": { \"signingContext\": { \"networkType\": \"DEVNET\", \"signatureType\": \"TRANSACTION_PUBLIC\" }, \"rawBytes\": \"c2lnbmF0dXJl\" } } }";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 440 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979598)